src/components/ListItems.js

```diff
@@ -0,0 +1,75 @@
+/* eslint-disable no-nested-ternary */
+import { useSelector } from 'react-redux';
+import { CircularProgress } from 'react-cssfx-loading';
+import { Link } from 'react-router-dom';
+import { Input } from 'semantic-ui-react';
+import { useState } from 'react';
+
+const ListItems = () => {
+  const countries = useSelector((state) => state.covidReducer.oneCountry);
+  const [filteredResults, setFilteredResults] = useState([]);
+  const [searchInput, setSearchInput] = useState('');
+  const searchItems = (searchValue) => {
+    setSearchInput(searchValue);
+    if (searchInput !== '') {
+      const filteredData = countries.filter((item) => Object.values(item).join('').toLowerCase().includes(searchInput.toLowerCase()));
+      setFilteredResults(filteredData);
+    } else {
+      setFilteredResults(countries);
+    }
+  };
+  const populate = countries
+    ? countries.map((country) => (
+      <div key={country.id} className="country d-flex">
+        <Link
+          className="navLink"
+          to={{
+            pathname: `/itemsdetails/${country.name}`,
+          }}
+        >
+          <h4 className="companyName">{country.name}</h4>
+        </Link>
+        <div>
+          <p className="casesConfirmedVal">{country.confirmedCases}</p>
+          <p className="casesConfirmed">cases confirmed</p>
+        </div>
+      </div>
+    ))
+    : <div className="spinLoad"><CircularProgress className="BarWave" color="#fff" width="20%" duration="2s" /></div>;
+  return (
+    <div className="list-container">
+      <h2>Search by Country name</h2>
+      <Input
+        className="search"
+        icon="search"
+        placeholder="Search..."
+        onChange={(e) => searchItems(e.target.value)}
+      />
+      <div className="list-countries d-flex">
+        {searchInput.length > 1
+          ? (
+            filteredResults.map((country) => (
+              <div key={country.id} className="country d-flex">
+                <Link
+                  className="navLink"
+                  to={{
+                    pathname: `/itemsdetails/${country.name}`,
+                  }}
+                >
+                  <h4 className="companyName">{country.name}</h4>
+                </Link>
+                <div>
+                  <p className="casesConfirmedVal">{country.confirmedCases}</p>
+                  <p className="casesConfirmed">cases confirmed</p>
+                </div>
+              </div>
+            ))
+          )
+          : populate}
```

Comment:
Let's stick to the design guide as much as possible ✊ 

Your home page
>![Screenshot (351)](https://user-images.githubusercontent.com/66112760/147087942-281b3003-1137-40f9-b20d-1871fc53f652.png)

Design Guideline home page
>![Screenshot (347)](https://user-images.githubusercontent.com/66112760/147088046-9a7bbbae-3345-4177-9c70-a44b586a543e.png)